src/celesto/__init__.py

```
@@ -1,8 +1,8 @@
 """Celesto SDK package."""
 
 from .main import app
-from .sdk import Celesto
+from .sdk import Computer
 
 __version__ = "0.0.9"
 
-__all__ = ["app", "Celesto", "__version__"]
+__all__ = ["app", "Computer", "__version__"]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)